index.js

```diff
@@ -209,190 +229,222 @@ async function startBotPlayMatch(page, myCards, quest, claimQuestReward, priorit
     let username = await getElementText(page, '.dropdown-toggle .bio__name__display', 10000);
 
     if (username == process.env.ACCUSERNAME) {
-		misc.writeToLog('Already logged in!');
-	} else {
-		misc.writeToLog('Login')
-        await splinterlandsPage.login(page).catch(e=>{
+        misc.writeToLog('Already logged in!');
+    } else {
+        misc.writeToLog('Login')
+        await splinterlandsPage.login(page).catch(e => {
             misc.writeToLog(e);
             throw new Error('Login Error');
         });
     }
-	
-	await waitUntilLoaded(page);
-	const erc = (await getElementTextByXpath(page, "//div[@class='dec-options'][1]/div[@class='value'][2]/div", 100)).split('.')[0];
-	misc.writeToLog('Current Energy Capture Rate is ' + erc + "%");
-	if (parseInt(erc) < ercThreshold) {
-		misc.writeToLog('ERC is below threshold of ' + ercThreshold + '% - skipping this account');
-		return;
-	}
-	await page.waitForTimeout(1000);
+    await waitUntilLoaded(page);
+    const erc = parseInt((await getElementTextByXpath(page, "//div[@class='dec-options'][1]/div[@class='value'][2]/div", 100)).split('.')[0]);
```

Comment:
ecr is available at this api: https://game-api.splinterlands.io/players/details?name=$player

I'm quite sure making api call without the browser is way cheaper